storage/src/main/java/org/commonjava/storage/pathmapped/core/PathMappedFileManager.java

```diff
@@ -71,8 +70,7 @@ public PathMappedFileManager( PathMappedStorageConfig config, PathDB pathDB, Phy
             logger.info( "Start path-mapped GC thread, gcIntervalInMinutes: {}", gcIntervalInMinutes );
             int priority = 4;
             String name = "path-mapped-gc";
-            gcThreadPool = Executors.newScheduledThreadPool( 1, new NamedThreadFactory( name, new ThreadGroup( name ),
-                                                                                        true, priority ) );
+            gcThreadPool = Executors.newScheduledThreadPool( 1 );
```

Comment:
Do we still need to identify the thread by the name ? if then, we can copy the NamedThreadFactory from weft.

Reply:
I guess not need. I think we will disable the gc thread from indy and only keep it in the store service. We only need one gc working anyway.

Reply:
See, LGTM.